dask/tests/test_layers.py

```diff
@@ -250,3 +251,41 @@ def test_dataframe_cull_key_dependencies(op):
     culled_graph = graph.cull(result.__dask_keys__())
 
     assert graph.get_all_dependencies() == culled_graph.get_all_dependencies()
+
+
+@pytest.mark.parametrize("op", [_shuffle_op, _groupby_op])
```

Comment:
It doesn't look to me like `op` is used here

Reply:
oops!